### PR TITLE
feature/FOUR-9496: [Story]Designer: Show the Projects

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
As a Designer I want to have a quick access to Recently Opened Projects.
image-20231002-153100

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-9496

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-7efc34d7db.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-7efc34d7db.eng.processmaker.net

File: resources/js/processes/designer/MyProject.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
         </b-nav-item>
       </b-navbar-nav>
     </b-navbar>
-    <div class="container">
+    <div v-if="false" class="container">
```
Comment (fagubla) [inline resources/js/processes/designer/MyProject.vue:16]: why???

File: resources/js/processes/designer/ProcessesLastModifiedListing.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        <b-link href="#">
+          {{ $t("Create a Project") }}
+        </b-link>
+      </div>
+    </div>
+  </div>
+  <div v-else class="data-table">
+    <data-loading
+      v-show="shouldShowLoader"
+      :for="/\/processes\?page/"
```
Comment (fagubla) [inline resources/js/processes/designer/ProcessesLastModifiedListing.vue:23]: this is not closed correctly

File: resources/js/processes/designer/ProcessesLastModifiedListing.vue [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        :css="css"
+        :api-mode="false"
+        :fields="fields"
+        :data="data"
+        data-path="data"
+        pagination-path="meta"
+        :no-data-template="$t('No Data Available')"
+      >
+        <template slot="title" slot-scope="props">
+          <a v-bind:href="`/designer/projects/${props.rowData.id}`" v-uni-id="props.rowData.id.toString()">{{props.rowData.title}}
```
Comment (fagubla) [inline resources/js/processes/designer/ProcessesLastModifiedListing.vue:45]: the value must go in a new line

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes/designer/ProcessesLastModifiedListing.vue [hunk 4]
```diff
@@ -0,0 +1,209 @@
+<template>
+  <div v-if="!existData" class="container">
```
Comment (fagubla) [inline resources/js/processes/designer/ProcessesLastModifiedListing.vue:2]: What happen if existData is an empty array ?

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-7efc34d7db.eng.processmaker.net

File: resources/js/processes/designer/ProcessesLastModifiedListing.vue [hunk 5]
```diff
@@ -0,0 +1,209 @@
+<template>
+  <div v-if="!data" class="container">
```
Comment (fagubla) [inline resources/js/processes/designer/ProcessesLastModifiedListing.vue:2]: data.length === 0

File: resources/js/processes/designer/ProcessesLastModifiedListing.vue [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+          sortField: "updated_at",
+          callback: "formatDate",
+        },
+        {
+          name: "__slot:actions",
+          title: "",
+        },
+      ],
+      actions: [
+        { value: "open-item", content: "Open", link: true, href: "/designer/projects/{{id}}", icon: "fas fa-sign-in-alt"},
```
Comment (fagubla) [inline resources/js/processes/designer/ProcessesLastModifiedListing.vue:109]: fix the code style like fields

File: resources/js/processes/designer/ProcessesLastModifiedListing.vue [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+import EllipsisMenu from "../../components/shared/EllipsisMenu.vue";
+
+const uniqIdsMixin = createUniqIdsMixin();
+
+export default {
+  components: {
+    EllipsisMenu,
+  },
+  mixins: [datatableMixin, dataLoadingMixin, uniqIdsMixin],
+  props: ["status", "permission", "isDocumenterInstalled", "currentUserId"],
```
Comment (fagubla) [inline resources/js/processes/designer/ProcessesLastModifiedListing.vue:80]: what is the purpose of these props?

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-7efc34d7db.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-7efc34d7db.eng.processmaker.net

**Pull request merged**